qiskit_metal/_gui/widgets/create_component_window/model_view/tree_model_param_entry.py

```diff
@@ -457,7 +457,21 @@ def delete_node(self, cur_index: QModelIndex):
 
         # remove from parent
         if (cur_node.name, cur_node) in cur_node.parent.children:
+
             cur_node.parent.children.remove((cur_node.name, cur_node))
+
+            # cleaning up dangling branches
+            parent = old_parent = cur_node.parent
+            if len(parent.children) < 1:
+                while len(parent.children) < 2:
+                    old_parent = parent
+                    parent = old_parent.parent
+                    old_parent.children = []
+
+                if len(old_parent.children) < 1:
+                    print(f"removing {old_parent.name} from {parent.name}")
```

Comment:
Was this part of testing your code, do we need the print statement?

Reply:
fixed! Thanks for pointing that out!